### DISCUSSION TITLE
Compare to https://github.com/abrochard/kubel

### DISCUSSION THREAD

**Discussion by TxGVNN** [eshelyaron/kubed#4] [Q&A]
Hi, today I see your package. Looks amazing.

I wonder what is different with kubel package. I used this package a long-time ago and it is very good.
Have you ever tried kubel?

**Comment by eshelyaron** [comment]
Hi @TxGVNN,

> Hi, today I see your package. Looks amazing.

Thank you!

> I wonder what is different with kubel package. I used this package a long-time ago and it is very good.

I think this is similar to #3, so my answer there (https://github.com/eshelyaron/kubed/discussions/3#discussioncomment-10285159) mostly applies here as well.
I've also elaborated a bit about this topic here https://eshelyaron.com/posts/2024-08-09-kubed-is-now-on-gnu-elpa.html.

The bottom line is that I would love to hear from you, as a `kubel` user, what you perceive as the most notable differences.

>  Have you ever tried kubel?

I have, and I agree that it's quite good, I think the author and contributors behind `kubel` are doing a great job.
For one, `kubel` has very nice support for showing resource tables for arbitrary resource types.  In Kubed, you can only list supported resource types, although it's easy to add support for more resources (see https://eshelyaron.com/kubed.html#Extending-Kubed).
On the other hand, Kubed has some features that `kubel` lacks, such as support for _bookmarking_ resource lists, exploring documentation of resource types and their fields, a global menu-bar menu and contextual right-click menus, and the ability to display the details of a specific resource directly without going through the list buffer.
